### DISCUSSION TITLE
Using Windows file tags for organization

### DISCUSSION THREAD

**Discussion by Japhoto** [tfeldmann/organize#355] [Q&A]
Is there a way to use tags in Windows like there is with "macos_tags"? Maybe a workaround if I'm not totally blind and just couldn't find the feature :)

I'd like to use a combination of filters and tags to create automatic file sorting to a quite complex file hierarchy. It means manually tagging the files, but I see no other way to differentiate say PDF-files "by use case" otherwise.

**Comment by Japhoto** [comment]
I think I answered myself here, since Windows tags can't be added to all files. I'll experiment with adding labels & tags on my work Mac and see if that works for my use case :)

**Reply by tfeldmann** [reply]
I didn't even know windows has something like tags! Maybe the `exif` filter can read them?
What do you think about having tags inside the filename, like `My_Photo #vacation.jpg`? This is something proposed in the very first issue #1

**Reply by Japhoto** [reply]
Hey Thomas,

Windows does indeed have tags, but it's a very convoluted and non-inclusive system. All my photos are in Lightroom, music files are being handled by Beets, so that basically leaves the rest of the file formats. Most of which Windows tags won't work with 😄

I ended up making a "Sortbox"-folder on my desktop, copied only the existing directory structure from my Truenas server under that folder and have been sorting my files in those directories (along new ones). After that I've been able to run Organize on that folder with "subfolder: true" and move with {relative_path} at the end of the destination.

This empties out the folders, but leaves the directory structure intact for future files.

To be clear, I only need to do this bigger batch once, combining years worth of old files into a reasonable directory structure and be done with it.

I might also make a deduplication script since there's going to be duplicates of the same files once I get further along 😅
